microsoftgraph/microsoft-graph-explorer
# Enable auto setting the content-type to json when selecting PUT

@ailae

## Overview

Enables auto set of content-type header when selecting PUT verb

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Login with any account
* Select PUT verb
* Check the request http headers
